packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py

```diff
@@ -615,7 +615,7 @@ def remove_user_from_org(user_id: int):
     with get_api_client() as api_client:
         try:
             (page, _) = api_client.users_api.list(
-                filter='{"==":[{"var":"id"},"%s"]}' % (user_id,),
+                filter=f'{{"==":[{{"var":"id"}},"{user_id}"]}}',
```

Comment:
I can't say it's easier to read now after the change. The {} are a part of the syntax here in the filter expression, so it makes total sense not to use f-strings here.